src/components/OSCALSystemCharacteristics.js

```diff
@@ -56,7 +56,7 @@ export default function OSCALSystemCharacteristics(props) {
             </Grid>
             <Grid
               item
-              xs={5}
+              xs={6}
```

Comment:
https://mui.com/api/grid/

We could also do `xs={auto}` right?
Does that look better?

Reply:
Umm I guess we could? But nothing else here really does so I didn't want to get too crazy with it in a bug fix.

Reply:
Might make a good `enhancement` ticket?